Microsoft.CodeAnalysis.Metrics.targets needs to wrap arguments to metrics.exe in quotation chars

**NuGet Package**: [Microsoft.CodeAnalysis.Metrics](https://www.nuget.org/packages/Microsoft.CodeAnalysis.Metrics)
**Version**: 3.3.0 (Latest)

### Describe the bug

The Microsoft.CodeAnalysis.Metrics.targets file fails to wrap the arguments it passes to the Metrics.exe in quotation marks, causing the Metrics.exe to fail with an error similar to:

"C:\Users\<user>\Source\Repos\Calculator Solution\Calculator Solution\Calculator Solution.csproj" (Metrics target) (2) ->
(Metrics target) ->
  EXEC : error : File 'C:\Users\<user>\Source\Repos\Calculator' does not exist. [C:\Users\<user>\Source\Repos\Calculator Solution\Calculator Solution\Calculator Solution.csproj]
  C:\Users\<user>\Source\Repos\Calculator Solution\packages\Microsoft.CodeAnalysis.Metrics.3.3.0\build\Microsoft.CodeAnalysis.Metrics.targets(15,5): error MSB3073: The command
 ""C:\Users\<user>\Source\Repos\Calculator Solution\packages\Microsoft.CodeAnalysis.Metrics.3.3.0\build\\..\Metrics\Metrics.exe" /project:C:\Users\<user>\Source\Repos\Calculator
 Solution\Calculator Solution\Calculator Solution.csproj /out:Calculator Solution.Metrics.xml" exited with code 2. [C:\Users\<user>\Source\Repos\Calculator Solution\Calculator
 Solution\Calculator Solution.csproj]

    0 Warning(s)
    2 Error(s)

### Steps To Reproduce

Create a new C# solution that has spaces in it's name similar to the above.
Add the Microsoft.CodeAnalysis.Metrics nuget package to the project
Build the project with /t:Metrics

### Expected behavior

The expected behavior is that the Metrics.exe is successfully run against the specified project

### Actual behavior

The actual behavior is the error message as noted in the bug description above.

## Additional context
The cause of this issue is in the Microsoft.CodeAnalysis.Metrics.target file, and can be fixed by adding quote characters around the arguments the Metrics.exe application is being invoked with. For example:
```
  <Target Name="Metrics">
    <Exec Command='&quot;$(MetricsExeFolder)\$(MetricsExeName).exe&quot; /project:&quot;$(MSBuildProjectFullPath)&quot; /out:&quot;$(MetricsOutputFile)&quot;'
          WorkingDirectory="$(MSBuildProjectDirectory)">
    </Exec>
  </Target>
```

--- a/nuget/Microsoft.CodeAnalysis.Metrics/Microsoft.CodeAnalysis.Metrics.targets
+++ b/nuget/Microsoft.CodeAnalysis.Metrics/Microsoft.CodeAnalysis.Metrics.targets
@@ -12,7 +12,7 @@
   </PropertyGroup>
 
   <Target Name="Metrics">
-    <Exec Command='&quot;$(MetricsExeFolder)\$(MetricsExeName).exe&quot; /project:$(MSBuildProjectFullPath) /out:$(MetricsOutputFile)'
+    <Exec Command='&quot;$(MetricsExeFolder)\$(MetricsExeName).exe&quot; /project:&quot;$(MSBuildProjectFullPath)&quot; /out:&quot;$(MetricsOutputFile)&quot;'
           WorkingDirectory="$(MSBuildProjectDirectory)">
     </Exec>
   </Target>

